### ISSUE TITLE
rst markup bugs in docs/

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6978]
|<img alt="reverland's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| reverland reported|
|-|-|
|Trac ID|trac#6978|
|Type|defect|
|Created|2014-02-19 16:39:53Z|
|Branch|https://github.com/twisted/twisted/tree/docs-rst-bugs-6978|

It looks many of the code blocks in the .rst files are not properly parsed.

For example, you can find this:

    If the cancelled Deferred is waiting on another``Deferred``

The problem may be identified by regular expression like \w``\w ? 

Attachments:

* [docs-rst-bugs-6978-1.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/fc4/fc498482719af6cbda5c2c65cd198d2632178017/a75aa8f9f4648c502bc7633c3589b7af05e83195.diff) (12605 bytes) - added by reverland on 2014-02-19 17:43:45Z - Just learn to file a patch....

<details><summary>Searchable metadata</summary>

```
trac-id__6978 6978
type__defect defect
reporter__reverland reverland
priority__normal normal
milestone__None None
branch__branches_docs_rst_bugs_6978 branches/docs-rst-bugs-6978
branch_author__adiroiban adiroiban
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1392827993000000 1392827993000000
changetime__1447939782513803 1447939782513803
version__None None
owner__reverland reverland
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="reverland's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_1"></a>|reverland commented|
|-|-|

Sorry for this poorly written ticket

docs/projects/web/howto/using-twistedweb.rst :  507
docs/projects/web/howto/using-twistedweb.rst :  560
docs/projects/web/howto/web-in-60/static-content.rst :  116
docs/projects/core/howto/application.rst :  124
docs/projects/core/howto/application.rst :  171
docs/projects/core/howto/application.rst :  219
docs/projects/core/howto/endpoints.rst :  46
docs/projects/core/howto/endpoints.rst :  83
docs/projects/core/howto/defer.rst :  722
docs/projects/core/howto/defer.rst :  730
docs/projects/core/howto/defer.rst :  767
docs/projects/core/howto/defer.rst :  778
docs/projects/core/howto/defer.rst :  784
docs/projects/core/howto/defer.rst :  824
docs/projects/core/howto/defer.rst :  1047
docs/projects/core/howto/trial.rst :  1039
docs/projects/core/howto/trial.rst :  1049
docs/projects/core/howto/trial.rst :  1050
docs/projects/core/development/policy/test-standard.rst :  498
docs/projects/core/development/policy/test-standard.rst :  531

**Comment by twisted-trac** [comment]
|<img alt="reverland's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_2"></a>|reverland commented|
|-|-|

Thanks for exarkun has pointed out that in irc.....

However, I've tidied it up...now...

[00:34] <exarkun> You could also mention some context around where the error occurs or the line number.


```
docs/projects/web/howto/using-twistedweb.rst :  507
docs/projects/web/howto/using-twistedweb.rst :  560
docs/projects/web/howto/web-in-60/static-content.rst :  116
docs/projects/core/howto/application.rst :  124
docs/projects/core/howto/application.rst :  171
docs/projects/core/howto/application.rst :  219
docs/projects/core/howto/endpoints.rst :  46
docs/projects/core/howto/endpoints.rst :  83
docs/projects/core/howto/defer.rst :  722
docs/projects/core/howto/defer.rst :  730
docs/projects/core/howto/defer.rst :  767
docs/projects/core/howto/defer.rst :  778
docs/projects/core/howto/defer.rst :  784
docs/projects/core/howto/defer.rst :  824
docs/projects/core/howto/defer.rst :  1047
docs/projects/core/howto/trial.rst :  1039
docs/projects/core/howto/trial.rst :  1049
docs/projects/core/howto/trial.rst :  1050
docs/projects/core/development/policy/test-standard.rst :  498
docs/projects/core/development/policy/test-standard.rst :  531

```

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra commented|
|-|-|

Putting it back up for review.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_4"></a>|@adiroiban commented|
|-|-|

(In [46272]) Branching to docs-rst-bugs-6978.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_5"></a>|@adiroiban set owner to @thijstriemstra|
|-|-|

Thanks for the patch.

Please note that the patch no longer applies on trunk.

I have manually inspected it and applied the required changes on trunk.

Some changes were already made.

thijs, please take a look and let me know if all is ok.

Thanks!

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_6"></a>|@thijstriemstra set owner to @adiroiban|
|-|-|

Looks good, please merge. Thanks!

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_7"></a>|@adiroiban set owner to reverland|
|-|-|

Thanks for the review.

Hope the tests are done... and will merge.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_8"></a>|@adiroiban set status to `closed`|
|-|-|

(In [46287]) Merge docs-rst-bugs-6978: Fix markup bugs in docs.

Author: reverland
Reviewer: adiroiban, thijs
Fixes: [#6978](https://github.com/twisted/twisted/issues/6978)
